api/answers/views.py

```diff
@@ -0,0 +1,211 @@
+"""
+Views operations for the answers
+"""
+from django.shortcuts import get_object_or_404
+from django.core.exceptions import ValidationError
+
+from rest_framework.views import APIView, Response
+from rest_framework.request import Request
+from rest_framework import status, permissions
+from rest_framework.permissions import AllowAny, IsAuthenticated
+from rest_framework.decorators import permission_classes, api_view
+
+from .models import Answer, AnswerVote
+from questions.models import Question
+from meetups.models import Meetup
+from .serializers import AnswerSerializer
+
+
+# Create your views here.
+
+
+class AnswersPostView(APIView):
```

Comment:
Hello @matthenge ,
@Bakley  coded this endpoint for post answer. I can see you have created it too...Did you sync with him on this?

Reply:
yeah I just needed something to work with